interfaces/schemas/quiz.json

```diff
@@ -1 +1,106 @@
splay in the html `textarea` or `input`","examples":["Type your text here..."],"type":"string"},"keywords":{"description":"List of keywords that the answers should contain. It is up to the developer to implement if the answer should contain ALL or ANY of the keywords.","examples":["bytes","octets"],"type":"array","items":{"type":"string"}}}},"Quizoot.UploadQuestion":{"type":"object","properties":{"files":{"description":"List of files to upload.","type":"array","items":{"$ref":"#/definitions/Quizoot.File"}},"max_size":{"description":"Maximum size of each files. Can be expressed in bytes, megabytes (MB), etc. See TODO for valid units.","examples":["10MB"],"type":"string"},"max_files":{"description":"Maximum number of files to upload.","default":1,"minimum":1,"type":"number"}},"required":["files","max_files","max_size"]},"Quizoot.File":{"type":"object","properties":{"type":{"$ref":"#/definitions/Quizoot.FileType","description":"File type enum. Can be `.txt`, `.csv`, `.pdf`, etc. See {@link FileType}."},"content":{"description":"Encoded `base64` string of the file's content.","type":"string"}},"required":["content","type"]},"Quizoot.FileType":{"description":"Enum of supported file extensions.","enum":[".csv",".img",".jpeg",".jpg",".pdf",".png",".txt"],"type":"string"},"Quizoot.CodeQuestion":{"type":"object","properties":{"content":{"description":"Code content. This will be run using a `python3` runner and evaluated.","type":"string"},"language":{"description":"Coding language. Only `python3` is supported.","type":"string","enum":["python3"]},"expected_output":{"description":"Expected output after running the code.","type":"string"}},"required":["content","expected_output","language"]},"Quizoot.Grading":{"type":"object","properties":{"point_value":{"description":"Specifies how much a correct answer count for in the total score.","type":"number"},"feedback":{"$ref":"#/definitions/Quizoot.Feedback","description":"Feedback given after answering the question."}},"required":["feedback","point_value"]},"Quizoot.Feedback":{"type":"object","properties":{"explanation":{"description":"General feedback/explanation displayed whether the answer is wrong or correct.","type":"string"},"when_wrong":{"description":"Feedback when answer is wrong.","type":"string"},"when_right":{"description":"Feedback when answer is right.","type":"string"}},"required":["explanation"]},"Quizoot.Difficulty":{"description":"Question difficulty: easy, medium or hard.","enum":["EASY","HARD","MEDIUM"],"type":"string"},"Quizoot.Author":{"type":"object","properties":{"name":{"description":"The quiz author's first name.","type":"string"},"surname":{"description":"The quiz author's last name. Is optional.","type":"string"},"pseudo":{"description":"A pseudo for the quiz author.","type":"string"},"email":{"description":"The quiz author's email.","pattern":"^[a-zA-Z]+[\\w.\\d-]*@[a-zA-Z]+\\.[a-zA-Z]{1,3}$","type":"string"}},"required":["name"]}},"$schema":"http://json-schema.org/draft-07/schema#"}
```

Comment:
Hum I should probably fix this: `JSON.stringify(data, null, 2)` in order to pretty print the answer

Reply:
I can add it now